api.php

```diff
@@ -414,19 +407,15 @@ function getLatestReleases($modids) {
 			`release` 
 			join asset on (asset.assetid = `release`.assetid)
 			join (select max(created) as created, modid from `release` group by modid) latestrelease on (latestrelease.created = `release`.created)
-		where `release`.modid in (?)
+		where `release`.modidstr in ($modidParams) and `release`.modversion not in ($versionParams)

```

Comment:
I just realized there could be an issue where the query unintentionally filters out a version of a different mod because the version numbers match. Working on a fix

Reply:
Found a fix, which was to check against the combined mod+version rather than mod and version separately